src/helpers/helpers.core.js

```diff
@@ -39,7 +39,7 @@ export function isArray(value) {
     return true;
   }
   const type = Object.prototype.toString.call(value);
-  if (type.substr(0, 7) === '[object' && type.substr(-6) === 'Array]') {
+  if (type.startsWith('[object') && type.endsWith('Array]')) {
```

Comment:
```suggestion
  if (type.substring(0, 7) === '[Object]' && type.substring(type.length-6) === 'Array]') {
```
Bit better performance as starts and endWith

Reply:
Alright I removed the startsWith and endsWith usage